zipline/pipeline/factors/technical.py

```diff
@@ -144,6 +150,238 @@ def compute(self, today, assets, out, close, volume):
         out[:] = nanmean(close * volume, axis=0)
 
 
+class RollingPearsonOfReturns(SingleInputMixin, CustomFactor):
```

Comment:
cc @twiecki @CaptainKanuk for name bikeshedding here.

- Is the `Rolling` here necessary?
- Is `Against` a better preposition than `Of`?
- Should this be `PearsonR` ?
- Should this have the word `Correlation` and/or `Coefficient` somewhere?